docs/administration/deployment/upgrading-db/index.md

```diff
@@ -0,0 +1,87 @@
+# Upgrading database
+
+If you are the REANA cluster administrator upgrading an existing REANA
+cluster deployment from one REANA release series to another, such as
+from 0.7.* to 0.8.0, there might be changes in the REANA database schema
+between the two versions. In these cases, it is necessary to run a
+database upgrade script using [`alembic`](https://alembic.sqlalchemy.org/en/latest/).
+
+The `reana-db` command-line tool is provided to ease the database
+upgrade tasks. The tool is included in the `REANA-Server` component.
+After you upgrade REANA by running `helm upgrade` succcessfully, you can
+run the following command:
+
+```console
+$ kubectl exec -i -t deployment/reana-server -- reana-db alembic upgrade
+```
+
+The output will show the upgrades accomplished.
+
+For instance, this is an example of the command and its output:
+
+```console
+$ kubectl default exec -i -t deployment/reana-server -- reana-db alembic upgrade
+INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
+INFO  [alembic.runtime.migration] Will assume transactional DDL.
+INFO  [alembic.runtime.migration] Running upgrade  -> c912d4f1e1cc, Quota tables.
+INFO  [alembic.runtime.migration] Running upgrade c912d4f1e1cc -> ad93dae04483, Interactive sessions.
+INFO  [alembic.runtime.migration] Running upgrade ad93dae04483 -> 4801b98f6408, Job started and finished times.
+INFO  [alembic.runtime.migration] Running upgrade 4801b98f6408 -> f84e17bd6b18, Workflow complexity.
+INFO  [alembic.runtime.migration] Running upgrade f84e17bd6b18 -> 6568d7cb6710, storing full workflow workspace.
+```
+
+The `upgrade` command without any additional argument will attempt to
+upgrade the database to the latest revision. You can also pass a
+revision hash in the event that you just want to upgrade to a certain
+revision.
+
+Additionaly, you can check all the alembic `history` to list all the
```

Comment:
I think this "additionally" goes a bit counter the present story. The above story presented "usual upgrade" story, while this part starts "careful upgrade" story.

We have two options: 

- (a) introduce sections, since the page is getting long. The above could be entitled "**Quick upgrade procedure**" and the new one perhaps "**Detailed upgrade procedure** " and we start by explaining that if you want to have more control about the upgrade process, you start by checking which model version you have, the history, then do upgrade, then test and see, and if ever you'd like to downgrade this is how...

- (b)  We don't introduce sections and change the story to the "careful" one, as this will allow to explain everything smoothly in temporal progressive manner.  (E.g. CLI output won't "go back" as is the case now.)

WDYT? I guess former might be good, with first section being good for "quick impatient admins" and the section section may be good for those who'd like to understand more behind the scene.

Depending on which one we choos, the paragraph beginning "The `upgrade` command without any additional argument..." would move accordingly.

Reply:
I guess (b) makes more sense, I'll try something and push it.